parasoft-findings-teamcity-agent/src/main/java/com/parasoft/findings/teamcity/agent/ParasoftFindingsBuildProcess.java

```diff
@@ -253,6 +264,55 @@ private void reportUnexpectedFormat(File from) {
         _build.getBuildLogger().error("Unexpected report format: "+from.getAbsolutePath()+ ". \nPlease try recreating the report. If this does not resolve the issue, please contact Parasoft support.");
     }
 
+    private void parseAndSendViolationMessages(File pmdReport) {
+        String relativePath = _build.getCheckoutDirectory().toURI().relativize(pmdReport.toURI()).getPath();
+        String fileSize = new DecimalFormat("0.00").format(pmdReport.length()/1024f);
+        _build.getBuildLogger().message("Importing data from '"+relativePath+"' ("+fileSize+" KB) with 'message service' processor");
```

Comment:
Add a log info like the red box below.
![image](https://user-images.githubusercontent.com/109122571/225817891-838f11df-7f21-4c44-a01d-f948f0827f48.png)